ChameleonForms/FieldGenerators/Handlers/FieldGeneratorHandler.cs

```diff
@@ -89,58 +91,112 @@ protected FieldGeneratorHandler(IFieldGenerator<TModel, T> fieldGenerator)
         /// <inheritdoc />
         public abstract FieldDisplayType GetDisplayType(IReadonlyFieldConfiguration fieldConfiguration);
 
-        /// <inheritdoc />

```

Comment:
Realised that I had accidentally put inheritdoc on the protected static methods in error - they actually need xmldoc comments so added them while in there